### DISCUSSION TITLE
Γραφικό περιβάλλον σε ArchLinux

### DISCUSSION THREAD

**Discussion by p19tzam** [courses-ionio/hci#1595] [Announcements]
Καλησπέρα θα ήθελα να ρωτήσω αν μπορούμε να κάνουμε εγκατάσταση GUI(πχ kde) στο arch η το θέλετε απλός σε καθαρό τερματικό.

**Comment by epidrome** [comment]
για αυτό το μάθημα, δεν βαθμολογούμε το γραφικό περιβάλλον, γιατί δεν είναι αναγκαίο, ενημερώθηκαν και οι σχετικές οδηγίες

αν θέλετε, μπορείτε να δοκιμάσετε μερικά, και να δουλέψετε με το τερματικό τους την εργασία γραμμής εντολών,

σύμφωνα με την φιλοσοφία του archlinux τα kde, gnome, ονομάζονται περιβάλοντα επιφάνειας εργασίας και είναι (κακά) αντίγραφα των windows-macos, οι σοβαροί χρήστες βάζουν:
https://wiki.archlinux.org/title/window_manager

**Reply by g1f1** [reply]
Σωστά η φιλοσοφία του arch είναι το τερματικό καθώς βασίζεται στην χαμηλή κατανάλωση πόρων, το να βάλεις GUI σε arch καταστρέφεις ότι έχει χτίσει το distro.

"Μην ξεχνάτε, μετά το Unix και τον Linus τα Debian έδωσαν τα φώτα του πολιτισμού 😊💪"

**Reply by epidrome** [reply]
> "Μην ξεχνάτε, μετά το Unix και τον Linus τα Debian έδωσαν τα φώτα του πολιτισμού 😊💪"

υπάρχουν πολλά "ράμματα για την γούνα" του ανοιχτού κώδικα, που θα δούμε σε επόμενο μάθημα (τεχνολογία λογισμικού)
